hydra-access-controls/lib/hydra/config.rb

```diff
@@ -11,6 +11,8 @@ def []= key, value
           self.permissions = value
         when :user_model
           self.user_model = value
+        when :id_to_resource_uri
```

Comment:
can we just use `AF::Base.translate_id_to_uri=` see https://github.com/projecthydra/active_fedora/blob/ee566ed727b3b3d98dacb419c496911b468d66e6/spec/unit/core_spec.rb#L103